### PR TITLE
feat(labels): add namespace

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

File: pkg/util/templates/templates.go [hunk 1]
```diff
@@ -100,7 +99,7 @@ func sufficientProbes(windowSize, label string) string {
 }
 
 //render creates a monitoring rule for the defined multiwindow multi-burn rate alert
-func (r *multiWindowMultiBurnAlertRule) render(url, percent, label, alertName string) monitoringv1.Rule {
+func (r *multiWindowMultiBurnAlertRule) render(url, percent, label string, namespacedName types.NamespacedName, alertNameSuffix string) monitoringv1.Rule {
```
Comment (NautiluX) [inline pkg/util/templates/templates.go:102]: these are a looot of parameters, you think we could move some of them to the rule type?

**Comment by NautiluX** [comment]
/hold
let's fix the build in a separate PR (see #122 )

**Comment by NautiluX** [comment]
we should not rename the URL label in this PR. Let's think about and do it separately, as we need to take into account the impact this will have during the update of the operator on clusters.

File: pkg/util/templates/templates.go (hunk 1 above)
Comment (NautiluX) [reply pkg/util/templates/templates.go:100]: Glad you mention it! Addressed in the new commit.

**Comment by georgettica** [comment]
sure, I would be glad to fix this issue in a different PR :)

**Review by NautiluX — COMMENTED** [review]
/lgtm

**Pull request merged**